src/backend/base/langflow/components/brightdata/brightdata_structured_data.py

```
@@ -0,0 +1,1108 @@
+                "category": "Social Media",
+                "description": "Reddit posts, discussions, and community content",
+                "inputs": ["url"],
+                "defaults": {},
+                "domains": ["reddit.com"],
+                "url_patterns": [r"/r/", r"/comments/"],
+                "confidence_weight": 30,
+                "keywords": ["reddit", "post", "discussion"]
+            }
+        }
+
+    def _detect_website_type(self, url: str) -> Tuple[Optional[str], int, Dict[str, Any]]:
```

Ruff (0.11.9)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)